portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -41,6 +41,15 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     response.getWriter().println(json);
   }
 
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String userComment = request.getParameter("comment-input");
+    // Gets page the user sent the request from using a hidden form
```

Comment:
"Uses a hidden form to get the page the user sent the request from."

Reply:
Oops, grammar.